diff -Naur lua-5.3-orig/src/luaconf.h lua-5.3-long-double/src/luaconf.h
--- lua-5.3-orig/src/luaconf.h	2026-07-31 19:30:54.581392533 -0300
+++ lua-5.3-long-double/src/luaconf.h	2026-07-31 19:33:14.815107271 -0300
@@ -144,7 +144,7 @@
 #endif
 
 #if !defined(LUA_FLOAT_TYPE)
-#define LUA_FLOAT_TYPE	LUA_FLOAT_DOUBLE
+#define LUA_FLOAT_TYPE	LUA_FLOAT_LONGDOUBLE
 #endif
 
 /* }================================================================== */
