diff -Naur lua-5.5-orig/src/luaconf.h lua-5.5-float/src/luaconf.h
--- lua-5.5-orig/src/luaconf.h	2026-07-31 19:43:08.211255028 -0300
+++ lua-5.5-float/src/luaconf.h	2026-07-31 19:45:13.499649385 -0300
@@ -131,8 +131,8 @@
 
 
 /* Default configuration ('long long' and 'double', for 64-bit Lua) */
-#define LUA_INT_DEFAULT		LUA_INT_LONGLONG
-#define LUA_FLOAT_DEFAULT	LUA_FLOAT_DOUBLE
+#define LUA_INT_DEFAULT		LUA_INT_INT
+#define LUA_FLOAT_DEFAULT	LUA_FLOAT_FLOAT
 
 
 /*
