diff -Naur lua-5.3-orig/src/luaconf.h lua-5.3-float/src/luaconf.h
--- lua-5.3-orig/src/luaconf.h	2026-07-31 19:30:54.581392533 -0300
+++ lua-5.3-float/src/luaconf.h	2026-07-31 19:32:28.886333060 -0300
@@ -140,11 +140,11 @@
 ** default configuration for 64-bit Lua ('long long' and 'double')
 */
 #if !defined(LUA_INT_TYPE)
-#define LUA_INT_TYPE	LUA_INT_LONGLONG
+#define LUA_INT_TYPE	LUA_INT_INT
 #endif
 
 #if !defined(LUA_FLOAT_TYPE)
-#define LUA_FLOAT_TYPE	LUA_FLOAT_DOUBLE
+#define LUA_FLOAT_TYPE	LUA_FLOAT_FLOAT
 #endif
 
 /* }================================================================== */
