diff -Naur lua-5.4.8-orig/src/luaconf.h lua-5.4.8/src/luaconf.h
--- lua-5.4.8-orig/src/luaconf.h	2026-07-29 17:29:06.930920226 -0300
+++ lua-5.4.8/src/luaconf.h	2026-07-29 17:31:57.348606402 -0300
@@ -158,8 +158,13 @@
 #else		/* }{ */
 /* use defaults */
 
+#if !defined(LUA_INT_TYPE)
 #define LUA_INT_TYPE	LUA_INT_DEFAULT
+#endif
+
+#if !defined(LUA_FLOAT_TYPE)
 #define LUA_FLOAT_TYPE	LUA_FLOAT_DEFAULT
+#endif
 
 #endif				/* } */
 
