diff -Naur lua-5.4-orig/src/luaconf.h lua-5.4-float/src/luaconf.h
--- lua-5.4-orig/src/luaconf.h	2025-05-21 18:09:16.000000000 -0300
+++ lua-5.4-float/src/luaconf.h	2026-07-31 19:39:59.406073632 -0300
@@ -115,8 +115,8 @@
 
 
 /* Default configuration ('long long' and 'double', for 64-bit Lua) */
-#define LUA_INT_DEFAULT		LUA_INT_LONGLONG
-#define LUA_FLOAT_DEFAULT	LUA_FLOAT_DOUBLE
+#define LUA_INT_DEFAULT		LUA_INT_INT
+#define LUA_FLOAT_DEFAULT	LUA_FLOAT_FLOAT
 
 
 /*
